packages/i18n/data/zh-CN.json

```diff
@@ -1,310 +1 @@
-  },
-  "UIKit.RichTextBody.redoButtonLabel": {
-    "developer_comment": "Label for the redo button",
-    "string": "重做"
-  },
-  "UIKit.RichTextBody.styleDropdownLabel": {
-    "developer_comment": "Label for the style dropdown",
-    "string": "样式"
-  },
-  "UIKit.RichTextBody.styleDropdownOptionH1": {
-    "developer_comment": "Label for the `headline-one` option",
-    "string": "标题 H1"
-  },
-  "UIKit.RichTextBody.styleDropdownOptionH2": {
-    "developer_comment": "Label for the `headline-two` option",
-    "string": "标题 H2"
-  },
-  "UIKit.RichTextBody.styleDropdownOptionH3": {
-    "developer_comment": "Label for the `headline-three` option",
-    "string": "标题 H3"
-  },
-  "UIKit.RichTextBody.styleDropdownOptionH4": {
-    "developer_comment": "Label for the `headline-four` option",
-    "string": "标题 H4"
-  },
-  "UIKit.RichTextBody.styleDropdownOptionH5": {
-    "developer_comment": "Label for the `headline-five` option",
-    "string": "标题 H5"
-  },
-  "UIKit.RichTextBody.styleDropdownOptionParagraph": {
-    "developer_comment": "Label for the `paragraph` option",
-    "string": "段落"
-  },
-  "UIKit.RichTextBody.styleDropdownOptionPreformatted": {
-    "developer_comment": "Label for the `code` option",
-    "string": "预格式化"
-  },
-  "UIKit.RichTextBody.styleDropdownOptionQuote": {
-    "developer_comment": "Label for the `quote` option",
-    "string": "引用"
-  },
-  "UIKit.RichTextBody.underlinedButtonLabel": {
-    "developer_comment": "Label for the underline button",
-    "string": "下划线"
-  },
-  "UIKit.RichTextBody.undoButtonLabel": {
-    "developer_comment": "Label for the undo button",
-    "string": "撤销"
-  },
-  "UIKit.RichTextBody.unorderedListButtonLabel": {
-    "developer_comment": "Label for the bullet list button",
-    "string": "项目符号列表"
-  },
-  "UIKit.SearchSelectInput.loadingOptionsMessage": {
-    "developer_comment": "Text showed while the options are being loaded",
-    "string": "正在加载完全匹配项"
-  },
-  "UIKit.SearchSelectInput.noOptionsMessage": {
-    "developer_comment": "Text of search select dropdown when no options match search text",
-    "string": "找不到与您的搜索字词匹配的内容"
-  },
-  "UIKit.SearchSelectInput.placeholderMessage": {
-    "developer_comment": "Placeholder text for the input",
-    "string": "搜索方式…"
-  },
-  "UIKit.SelectInput.loadingOptions": {
-    "developer_comment": "Default input loading text",
-    "string": "正在加载..."
-  },
-  "UIKit.SelectInput.noOptionsMessageWithInputValue": {
-    "developer_comment": "Text of dropdown when no options match search text",
-    "string": "无选项"
-  },
-  "UIKit.SelectInput.noOptionsMessageWithoutInputValue": {
-    "developer_comment": "Text of dropdown when no options exist",
-    "string": "无选项"
-  },
-  "UIKit.SelectInput.placeholder": {
-    "developer_comment": "Default input placeholder text",
-    "string": "选择..."
-  },
-  "UIKit.TimeInput.placeholder": {
-    "developer_comment": "Placeholder for time input",
-    "string": "小时：分钟"
-  }
-}
+{}
```

Comment:
@mustafaasif2 @tdeekens I decided to keep this file for now with empty data.

The reason being is that in App Kit's `i18n` package the file is being referenced.

<img width="668" alt="image" src="https://github.com/user-attachments/assets/ca0f7dde-d0d3-4d17-9e16-38fa1877c526">

If we release UI Kit with the file removed, and App Kit is not updated to the latest version that removes the reference, there will be an error as the file is not found.

This way, we can release both UI Kit and App Kit independently without causing regressions.

Reply:
https://github.com/commercetools/ui-kit/pull/3025